python/ray/dag/compiled_dag_node.py

```diff
@@ -112,7 +115,27 @@ def do_exec_tasks(
             if done:
                 break
             for operation in schedule:
+                start_t = time.time()
```

Comment:
run this only when profiling is enabled. Also I recomend you to use time.perf_counter instead of time.time() https://stackoverflow.com/questions/52222002/what-is-the-difference-between-time-perf-counter-and-time-process-time

```python3
if enable_profiling:
    start_t = time.perf_counter()
```

Reply:
Thanks for the pointer! I'll try that out.

I did see that the timers of multiple workers on different nodes were not consistent with each other. Maybe this can be fixed by perf_counter.